app/Services/Manager.php

```diff
@@ -495,4 +495,27 @@ public function addUserToModel($model, $id, $user)
 
         return true;
     }
+
+    /**
+     * Fires off an event to send an email a user.
+     *
+     * @param  Gladiator\Models\User     $user
+     * @param  Gladiator\Models\Contest  $contest
+     */
+    public function sendEmail($user, $contest)
```

Comment:
is this a general email sender method now? Maybe we also pass in the `$type` and `$key` so it can send other messages, not just the welcome. if that's the case then we'll need the `$key` in there as well to differentiate between emails. This could also be a later update? Maybe note it?